include/battle_scripts.h

```diff
@@ -10,6 +10,7 @@ extern const u8 BattleScript_MakeMoveMissed[];
 extern const u8 BattleScript_PrintMoveMissed[];
 extern const u8 BattleScript_MoveMissedPause[];
 extern const u8 BattleScript_MoveMissed[];
+extern const u8 BattleScript_RemoveItemButItFailedAtkStringPpReduce[];
```

Comment:
I want to see if I can rename `BattleScript_ButItFailedAtkStringPpReduce` to `BattleScript_ButItFailedPpReduceAtkString` and if so, then it could be...
```suggestion
extern const u8 BattleScript_ButItFailedPpReduceAtkStringRemoveItem[];
```